### PR TITLE
Feature/append autosuggest [WEB-1388]

### PR DESCRIPTION
**Pull request by nikhiltri**
This PR appends "(deaccessioned)" to autosuggest requests. Additionally there is linting cleanup as part of these changes.

### CODE REVIEW

File: config/scout.php [hunk 1]
```diff
@@ -75,6 +116,24 @@
...
+    | source search engine with minimal configuration. Below, you can state
+    | the host and key information for your own MeiliSearch installation.
+    |
+    | See: https://docs.meilisearch.com/guides/advanced_guides/configuration.html
+    |
+    */
+
+    'meilisearch' => [
+        'host' => env('MEILISEARCH_HOST', 'http://localhost:7700'),
+        'key' => env('MEILISEARCH_KEY', null),
```
Comment (zachgarwood) [inline config/scout.php:134]: Do we need to add these to our local `.env`, or will these defaults work?

**Review by zachgarwood — APPROVED** [review]
Looks good, just have one question.

File: config/scout.php (hunk 1 above)
Comment (nikhiltri) [reply config/scout.php:134]: We're not using Mielisearch, so we don't need these values in our .env files. Added this to config just to have parity with the version of the package we're using.

**Review by web-dev-trev — APPROVED** [review]
LGTM! Thanks for figuring this one out Nikhil I was stuck on this for a bit 
Cat-stuck-paper-bag

**Pull request merged**